community-plugins.json

```diff
@@ -4272,5 +4272,12 @@
         "author": "Zsolt Viczian",
         "description": "ExcaliBrain is inspired by TheBrain and Breadcrumbs. It is an interactive, structured mind-map of your Obsidian Vault generated based on the folders and files in your Vault by interpreting the links, dataview fields, tags and YAML front matter in your markdown files.",
         "repo": "zsviczian/excalibrain"
+    },
+    {
+        "id": "obsidian-zotero",
+        "name": "Zotero",
```

Comment:
Sorry I didn't notice this earlier, but–

I think you might want to change the plugin name (something like Zotero Link is fine). Having it as just "Zotero" could imply it's an official product of the Zotero team and might cause a bit of confusion from users.

Reply:
Thank you for highlighting this, I've renamed it to ZoteroBridge here and in the repository now.